### ISSUE TITLE
Add a commit message

### ISSUE THREAD

**Issue by newswim** [carrot/ship#78]
It would be nice to have an ability to attach a commit message to the deployment. Specifically, this would be an option for gh-pages deployments.

**Comment by samccone** [comment]
+1

**Comment by jenius** [comment]
I can get down with that. Will take a look once I have a chance to really work on ship (soon, hopefully!)

**Comment by samccone** [comment]
yeah I need to drop some time into here as well


Will implement features for :pizza:

**Comment by jenius** [comment]
I will send you :pizza: or :dog2: for PRs

**Comment by samccone** [comment]
kthx

I will hold you to this


if not I will hold your :rabbit: for :pizza:

**Comment by jenius** [comment]
Fair enough. Right now my :rabbit: is eating :toilet_paper_tubes:, so I'm sure it would be happy to deliver :pizza: since it seems much more into :hay: and :cardboard:

**Comment by samccone** [comment]
:kissing:
